test/types/string/dmk/substring/substringBadRange2-codepoint.chpl

```diff
@@ -1,3 +1,7 @@
-var str = "åbçdèfghíjklm";
+var str1 = "abcdefghijklm";
 
-writeln("str[13:codepointIndex..(-1):codepointIndex] = ", str[13:codepointIndex..(-1):codepointIndex]);
+writeln("str1[13:byteIndex..(-1):byteIndex] = ", str1[13:byteIndex..(-1):byteIndex]);
+
+var str2 = "åbçdèfghíjklm";
+
+writeln("str2[13:codepointIndex..(-1):codepointIndex] = ", str2[13:codepointIndex..(-1):codepointIndex]);
```

Comment:
Not directly related to this test, but while I'm thinking of it:  Does slicing by unbounded byte or codepoint or int ranges work?  (e.g., `str2[12..]`)  Are there existing tests of it?

Reply:
Yes, we do have several tests that exercise that.  I had to make some new ones to make sure all the bases were covered.

Reply:
Found one after typing this, sorry (I find if I don't write questions when I think of them, I can't think of them again later).